__tests__/app.test.js

```diff
@@ -270,3 +269,47 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe(" GET /api/reviews/:review_id/comments", () => {
+  test("should return an array of comments for the given review_id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body;
```

Comment:
if you do decide to send it back in an object - this test would just need updating to be comments = body.comments